WEEKLY_UPDATES.md

```diff
@@ -1,5 +1,32 @@
 # Weekly Updates
 
+## 2023-12-14
+
+Hey there!
+
+This week's been mostly about delivering that last 1% of work before we can publish a release candidate. Other than that, we've also started planning work for past the stable release, so targeting the 1.7 release.
+
+The steering committee has also gathered (you can find the details [here](https://github.com/opentofu/opentofu/blob/main/TSC_SUMMARY.md#2023-12-11)) and planned the official date for the stable release to be the 10th of January.
+
+- Current Status and Up Next
+  - We've added GPG key signing to debian and rpm archives that we're distributing (thanks to BuildKite for sponsoring PackageCloud!)
+  - We've worked on changing the default provider namespace to `opentofu`, but ended up rolling that back, and (softly) deprecating the whole concept of namespace-less provider references altogether, due to the edge-cases involved.
+  - The registry now has an improved development workflow with a proper dev environment. It's also seen big runtime optimizations so refreshing providers and modules is now ~50% faster.
+  - We're working on the final documentation improvements before the stable release.
+  - We'll soon be starting work on 1.7 features.
+- How can I help?
+  - Since the beta release is out, right now the best way to help is to take the beta for a test drive and see if there are any bugs / issues.
+    - Most importantly, as mentioned above, please try running `tofu init` with your projects to double-check everything you use is available in the registry.
+  - **With the release candidate around the corner, it's the perfect time to start implementing OpenTofu support in 3rd party tooling!**
+  - Other than that, the best way to help is to create issues, discuss on issues, and spread the word about OpenTofu.
+  - There are some occasional minor issues which are accepted and open to external contribution, esp. ones outside the release-blocking path. We’re also happy to accept any minor refactors or linter fixes. [Please see the contributing guide for more details](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md).
```

Comment:
I would remove this to avoid the impression that we are happy about pure reformatting PRs and similar. Unfortunately, as a high profile project they are to be expected and will likely consume a lot of resources reviewing/re-testing.

```suggestion
  - There are some occasional minor issues which are accepted and open to external contribution, esp. ones outside the release-blocking path. [Please see the contributing guide for more details](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md).
```